app_spec/zomes/simple/code/src/lib.rs

```diff
@@ -51,10 +51,11 @@ pub fn handle_delete_my_link_with_tag(base: Address,target : String,tag:String)
 }
 
 
-pub fn handle_get_my_links(agent : Address,status_request:Option<LinksStatusRequestKind>) ->ZomeApiResult<GetLinksResult>
+pub fn handle_get_my_links(agent : Address,status_request:Option<LinksStatusRequestKind>,sort_order:Option<SortOrder>) ->ZomeApiResult<GetLinksResult>
```

Comment:
Looks like you're just following the existing paradigm, so no need to change, but I find this pretty hard to parse without spaces - maybe we should add these rust files to our `fmt` check as a separate code hygiene effort?

Reply:
I 100% agree! i dont know if it is because they are zomes but we should definitely look into this